### ISSUE TITLE
improve button contrast?

### ISSUE THREAD

**Issue by frehner** [blittle/racial-equity-banner#3]
<img width="688" alt="WCAG contrast ratio of colors, showing a ratio of 1.86 to 1 between the button color and the background color" src="https://user-images.githubusercontent.com/3054066/83440201-bbbefd00-a401-11ea-87ab-4fe0d014a3e3.png">

According to https://webaim.org/resources/contrastchecker/ the contrast ratio between the button color `#b1b1b1` and background color `#efefef` is 1.86:1

**Comment by blittle** [comment]
Agreed. I did it according to a design from a ux designer, but should have noticed it verified the ratio myself
